contrib/verifybinaries/README.md

```diff
@@ -1,41 +1,93 @@
 ### Verify Binaries
 
-#### Preparation:
+#### Preparation
 
-Make sure you obtain the proper release signing key and verify the fingerprint with several independent sources.
+As of Bitcoin Core v22.0, releases are signed by a number of public keys on the basis
+of the [guix.sigs repository](https://github.com/bitcoin-core/guix.sigs/). When
+verifying binary downloads, you (the end user) decide which of these public keys you
+trust and then use that trust model to evaluate the signature on a file that contains
+hashes of the release binaries. The downloaded binaries are then hashed and compared to
+the signed checksum file.
 
-```sh
-$ gpg --fingerprint "Bitcoin Core binary release signing key"
-pub   4096R/36C2E964 2015-06-24 [expires: YYYY-MM-DD]
-      Key fingerprint = 01EA 5486 DE18 A882 D4C2  6845 90C8 019E 36C2 E964
-uid                  Wladimir J. van der Laan (Bitcoin Core binary release signing key) <laanwj@gmail.com>
-```
+First, you have to figure out which public keys to recognize. Browse the [list of frequent
+builder-keys](https://github.com/bitcoin/bitcoin/tree/master/contrib/builder-keys) and
```

Comment:
Not sure if it makes sense to pick the trust based on this list. It is outdated and there are many keys, such as mine, which have never contributed a signature for any guix release (https://github.com/bitcoin/bitcoin/pull/26598#issuecomment-1337066475). Also, it doesn't seem ideal to blindly trust the list, when the only requirement to get on the list, is to upload one or two attestations.

It might be better (and less code) to just treat all good signatures as good, but untrusted, unless they are trusted by the local gpg.